examples/image-classification-web/README.md

```diff
@@ -0,0 +1,74 @@
+# Image Classification Web Demo Using Burn and WebAssembly
+
+## Overview
+
+This demo showcases how to execute an image classification task in a web browser using a model
+converted to Rust code. The project utilizes the Burn deep learning framework, WebGPU and
+WebAssembly . Specifically, it demonstrates:
+
+1. Converting an ONNX (Open Neural Networks Exchange) model into Rust code compatible with the Burn
+   framework.
+2. Executing the model within a web browser using WebGPU via the `burn-wgpu` backend and WebAssembly
+   through the `burn-ndarray` and `burn-candle` backends.
+
+## Running the Demo
+
+### Step 1: Build the WebAssembly Binary and Other Assets
+
+To compile the Rust code into WebAssembly and build other essential files, execute the following
+script:
+
+```bash
+./build-for-web.sh
+```
+
+### Step 2: Launch the Web Server
+
+Run the following command to initiate a web server on your local machine:
+
+```bash
+./run-server.sh
+```
```

Comment:
I think we should provide rust scripts instead at some point. It's hard to test on Windows otherwise.

Reply:
Yes, I agree! I started adding xtask just for that using xtask-wasm: https://github.com/rustminded/xtask-wasm
However, it was taking longer than I anticipated because we also need to translate the build script as well (with all compiler flags). xtask-wasm has its own build similar to wasm-pack which should work too. So instead I opted to file a ticket (still outstanding) to convert this project and MNIST as well.

With wasm-pack, one can build, watch project changes and run the server. We should take advatanage of it. 

@nathanielsimard what do you think about this plan?

Reply:
Good plan